ogr/abstract.py

```diff
@@ -437,35 +437,84 @@ def add_label(self, *labels: str) -> None:
         """
         raise NotImplementedError()
 
+    def get_statuses(self) -> List["CommitFlag"]:
+        """
+        Returns statuses for latest commit on pull request.
+
+        :return: [CommitFlag]
+        """
+        raise NotImplementedError()
+
+
+class CommitStatus(IntEnum):
```

Comment:
Do we need the comparison with `int`s?

What about this?
https://docs.python.org/3/library/enum.html#using-a-descriptive-string

Reply:
Had a look around abstract, I thought it's a convention, ok, I'll change it

Reply:
It allows some `string <-> enum` conversion.